deps/build.jl

```diff
@@ -4,6 +4,14 @@ using BinDeps
 
 using Compat
 
+# check for cairo version 
+function validate_cairo_version(name,handle)
+    f = Libdl.dlsym_e(handle, "cairo_version")
+    f == C_NULL && return false
+    v = ccall(f, Int32,())        
+    return v > 11000
```

Comment:
This would be a new restriction, right? I'll have to spin up a docker container and test for sure, but for example it looks like CentOS 6 is at Cairo 1.8.8, judging by https://rpmfind.net/linux/rpm2html/search.php?query=cairo.